locations/spiders/yettel_rs.py

```diff
@@ -0,0 +1,50 @@
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class YettelRSSpider(scrapy.Spider):
+    name = "yettel_rs"
+    item_attributes = {
+        "brand": "Yettel",
+        "brand_wikidata": "Q1780171",
+        "country": "RS",
+    }
+
+    def start_requests(self):
+        url = "https://www.yettel.rs/stores/latlong/"
+        yield scrapy.http.FormRequest(url=url, formdata={"initial": "true"}, method="POST")
```

Comment:
I don't think there is any data with that request. Try:
```suggestion
        yield scrapy.http.FormRequest(url=url, formdata={"lat": "0", "lng": "0", "dist": "10000000000"}, method="POST")
```